pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformFunction.java

```diff
@@ -225,6 +228,9 @@ public void initialize(Context context) {
         case "compute":
           transformStep = newComputeFieldFunction((ComputeConfig) step);
           break;
+        case "add-embeddings":
```

Comment:
what about `compute-ai-embeddings` ?

Reply:
actually if the is a compute kind of transformation how about leveraging the compute step and add more JSTL functions under a new `ai:` prefix?

For example, `ai:embed(input)` to invoke vector embeddings or `ai:complete()` to invoke the completion API. With that approach, you get all the compute features for free and the UX is consistent. We'd just need away to pass the authorization and the api endpoint to invoke.

Here is how the transformation could look like:

```
{
    "steps": [
        {
            "type": "compute",
            "fields": [
                {
                    "name": "value.vectorizedText",
                    "type": "STRING",
                    "expression": "ai:embed(value.text)"
                }
            ]
        }
    ]
}
```

Reply:
I think that it is better to have a new step, because it will make the feature more visible to users